### PR TITLE
Move to Train branch

### PR DESCRIPTION
**Pull request by elfosardo**
Moving to train branch

### CODE REVIEW

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1083/

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1086/

**Comment by metal3ci** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/1088/

**Comment by russellb** [comment]
metal3ci should probably be turned off on this repo.  That CI tests OpenShift, which uses Ironic images from OpenShift, and not this one.  @derekhiggins

**Comment by derekhiggins** [comment]
> metal3ci should probably be turned off on this repo. That CI tests OpenShift, which uses Ironic images from OpenShift, and not this one. @derekhiggins

I was hoping to use these images for CI, see https://github.com/openshift-metal3/dev-scripts/pull/759

**Comment by hardys** [comment]
> > metal3ci should probably be turned off on this repo. That CI tests OpenShift, which uses Ironic images from OpenShift, and not this one. @derekhiggins
> 
> I was hoping to use these images for CI, see [openshift-metal3/dev-scripts#759](https://github.com/openshift-metal3/dev-scripts/pull/759)

Yeah it would be nice to have the option of using these upstream images in dev-scripts IMO, and your locally built PRs seem a simple way to enable that.

**Comment by elfosardo** [comment]
let's hold on to this, the RDO stable/train branch should be available and we actually need that

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM docker.io/centos:centos7
 
 RUN yum install -y python-requests && \
-    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - -b stein current-tripleo && \
+    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - -b train current-tripleo && \
```
Comment (hardys) [inline Dockerfile]: The commit doesn't match the commit - master isn't a stable branch, so maybe say "Switch to train OpenStack release" or something?

**Comment by elfosardo** [comment]
@hardys I thought the stable/train branch was not available yet in RDO, but it is
we can merge this at the earliest convenience

File: Dockerfile (hunk 1 above)
Comment (elfosardo) [reply Dockerfile]: odd, I'm pretty sure I amended it
going to push one more time

**Comment by hardys** [comment]
I discussed this with @elfosardo yesterday and we realized that while the RDO repos are ready, the images (which we need for ironic-ipa-downloader) aren't, and we're still a couple of weeks from the final train release.

Since it seems likely that upstream metal3 will continue to consume new features from Ironic, I proposed on the metal3-dev ML that we instead switch to master, so far the consensus seems to be that we should but opinions welcome on that.   https://groups.google.com/forum/#!topic/metal3-dev/wdAVjp44kMY

**Pull request merged**